src/azure-cli/HISTORY.rst

```diff
@@ -49,8 +49,8 @@ Release History
 
 **Reservations**
 
-* Upgrading SDK Version to 0.6.0
-* Add billingplan details info after calling Get-Gatalogs
+* Add new command `az reservations reservation-order calculate` to calculate the price for a reservation
```

Comment:
Why do you  delete previous release notes here?

Reply:
I think that one was not released yet. After this PR the main change should be adding those two new commands. That's what I think.

Reply:
should I add it back?